src/main/scala/io/gitlab/agluszak/tickets/services/ScreeningService.scala

```diff
@@ -0,0 +1,25 @@
+package io.gitlab.agluszak.tickets.services
+
+import java.time.LocalDateTime
+
+import io.gitlab.agluszak.tickets.PostgresProfile.api._
+import io.gitlab.agluszak.tickets.repositories.ScreeningRepository
+import io.gitlab.agluszak.tickets.{MovieScreenings, ScreeningDetails, ScreeningId}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class ScreeningService(db: Database, screeningRepository: ScreeningRepository) {
+  def listScreeningsInInterval(from: LocalDateTime, to: LocalDateTime): Future[Seq[MovieScreenings]] =
+    db.run(
```

Comment:
your service logic (core logic) is tightly related to db. Do you see any downsides of current solution?